containers/cpp-mariadb/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "C++ and MariaDB",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"ms-vscode.cpptools",
+		"ms-vscode.cmake-tools"
```

Comment:
How about adding : `ms-vscode.cpptools-extension-pack` . Seems like useful for debugging.

Reply:
Good call, thank you!